content/300-guides/150-testing/150-integration-testing.mdx

```diff
@@ -294,9 +294,9 @@ Before the test suite is run the database is seeded with data. After the test su
 
 <Tip>
 
-  Using `deleteMany` may suffice in situations where you know ahead of time how your schema is structured. This is because the operations need to be executed in the correct order according to how the model relations are setup.
+Using `deleteMany` may suffice in situations where you know ahead of time how your schema is structured. This is because the operations need to be executed in the correct order according to how the model relations are setup.
 
-  However, this doesn't scale as well as having a more generic solution that maps over your models and performs a truncate on them. For those scenarios and examples of using raw SQL queries see [Deleting all data with raw SQL / `TRUNCATE`](../../concepts/components/prisma-client/crud#deleting-all-data-with-raw-sql--truncate)
+However, this doesn't scale as well as having a more generic solution that maps over your models and performs a truncate on them. For those scenarios and examples of using raw SQL queries see [Deleting all data with raw SQL / `TRUNCATE`](../../concepts/components/prisma-client/crud#deleting-all-data-with-raw-sql--truncate)
 
 </Tip>
```

Comment:
Would be really nice if it could format this as "html" and indent correctly.

Reply:
Looks like it doesn't allow to use 2 spaces

https://user-images.githubusercontent.com/4734297/124766861-bad79380-df50-11eb-9098-cba755e37568.mp4

Reply:
Does not really make sense though, right?
If this was a `<p>` or `<section>` it would probably put it behind 2 spaces?

Reply:
It's because of the spacing, if there are blank line after opening JSX tag, parser will treat text as markdown, and prepend it to the beginning ([example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAKgSwA4D4A6UABEQKoDOmUA5kQAYAmcANnPALICGUAnnUQFtOPIuQCuAMwmYwcIlVGYYYzjEzRyRAO4ALOACc5PCGKIBrKBC1FOezgyIQJRNQLk6rRY2P2iweoXlNchh9MTBlQwYAOiJ0HUxNRKIAIzgwTjFyORg9R2wDVXUoTSg4OAcYCFS5OAAPdLF4BwVcuUh9QwjHfSZfTjAOhipaKqIPazbBCCZmIkNmIo0bQ1E2MWxowhIACSs4ADcDABoXBM0GCDhyKAByGD9OVhtNLRY5zk0dTgORm2nVtQEAYZKIIMwmsUzqpBJxsJoIEdfN5fAIZixNNwHAV9BIIPoBJiXGEoBl4I5iG0BLEAGL4s4QbJ+BCcfTqTFQBz1TgCbCsBHOLJ-fScawAZQAigAZIgARzEIOuazkAG0ACIsNh-J5zBiqTjaJQ6eaioiSmUAeno6AASqQAHIAYQAgugAKJ0AC6AApotELX6LZBSXBsDByEGILzoAhwxbsGzyEIALRgZiYWNBsIMADEsy1NGTOuTepgnGTWiNyZFWmT5FlzGTydCYlJqjgAEpCKgLVg8IRu32CFBtmRKDR6PmONw+LCROIpDI5ApKMoliVtHpVt5zJZrLY4PZHM5XO5PCi-AEDckQmEIj4KrF4slkmkMlkcnkIDj16VypVqjSIh6kaZp5EpPIOi6B58T6GxBlgv4xgmM45DRWZ5hYX8VjkbJlE2AdiCIPY3iRU5cmSS5rjuB5yAyZ5Pm0d4XnGH5tQBOQgTKNkwDBCE1GgaEHiEeFHCRLwTFRdFmA5bEDDxAkiRbNtyUEqlaXpXJGVw2QoFZdkbE5YC6h5PklScIghQnGszWlOUFTZJVslVDVWDUCcdSIUsDUrXITXFOyrToW0HRdd0vV9f1A2DWQwwjSBozKWAIwTRIUzTDNYCzMRcynEYi2YRtvIrKsazrBsm2UslO0Insh0IEBjhAb8BJKZBQFZfQrAABVZBByGQEAni0YQBqalIRTAMw2DFbABhGZAWzgJq4AENIGCYBgpW4agVCBOkCVUdzqEGpwJEakAdBgARmAAdQSeByDm2QxX6pRMF+GAeEGsByDGkAqGyfQYG6kVqCEZAJCebImoAK3IOoACFJummAxR5OApSoOBIeh5aQHhuoxRGVgJTECB4FxmT8bm-QgcGgQGDqC7UtgW7MAYXJkAADgABiahNtNukVsEGhNrgMI4LvlCm4FB78BpQT5kzKCoKguwx5UwQxQc4cHOCpmGQGyARMEWsJ8fHahSfJymkCh6mmrLFJ2c5nRkAAJidkVMHTGhHSjCGUGuABWC6P3QTgUkVh2jYOBUAElOVjMUwDZMNnU5MUvtYQ24AAX3zoA))

Reply:
Hmmm weird that the empty lines influence it like that.